packages/deploy-configurations/configs/optimism.conf.ts

```diff
@@ -97,7 +97,7 @@ export const config: SystemConfig = {
       PositionCreated: {
         name: 'PositionCreated',
         deploy: false,
-        address: '0x8061c24823094E51e57A4a5cF8bEd3CCf09d316F',
+        address: '0x5743b5606e94fb534a31e1cefb3242c8a9422e5e',
```

Comment:
Well on optimism I cannot see any contract here.

Reply:
https://optimistic.etherscan.io/address/0x8061c24823094e51e57a4a5cf8bed3ccf09d316f#code

It's legitimately the PositionCreated contract. Address clash between AutomationBotV2 on mainnet and PC on optimism.

Reply:
Reverted changes to that file.